pkgs/by-name/wr/wrangler/package.nix

```
@@ -87,6 +101,12 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  versionCheckProgramArg = "-v";
+  doInstallCheck = true;
```

> Yes, but this ensures that the version is checked. Even if it's set by default. 

That's already taken care of by

```suggestion
  nativeInstallCheckInputs = [
    versionCheckHook
  ];
  doInstallCheck = true;
```

You only require `versionCheckProgramArg` if your program does not produce the version given one of the default arguments (`--help` or `--version`) and you therefore need to override those defaults.

> For me, this allows for better control;

However, you should just leave it as is now given your point that you like to be explicit.
Just wanted to make sure it was intentional. This comment is resolved.

---

Thanks for your help, I do appreciate being explicit and yes it is intentional.

---

Some programs like to print their full path somewhere in `--help`, which would make the check obsolete. So it's always a good idea to specify the argument.

---

It seems like `--version` should take precedence over `--help` to minimize the likelihood of this, but I agree it is ultimately best to just be explicit.